docs/en/writing-running-appium/android/android-appbundle.md

```diff
@@ -0,0 +1,51 @@
+## How To Test Android App Bundle
+
+Google has released [Android App Bundle](https://developer.android.com/platform/technology/app-bundle/) feature.
+`.aab` file is generated by the feature. We must upload the file to Google Play Store.
+
+We can get distributed apk files from the `.aab` file via [bundletool](https://github.com/google/bundletool). Using the generated files, we can test against release module. After [1](https://github.com/appium/appium-adb/pull/367) and [2](https://github.com/appium/appium-base-driver/pull/271), you can run Appium tests against `.apks` file. The commits are available over Appium 1.9.2.
```

Comment:
The support for apks is currently only present in UIA2

Reply:
, we can test against the release module.
...
Since Appium 1.9.2, you can run Appium tests against an `.apks` file.